pkg/fqdn/name_manager.go

```diff
@@ -66,13 +66,13 @@ func (n *NameManager) GetModel() *models.NameManager {
 // Lock must be held during any calls to RegisterForIdentityUpdatesLocked or
 // UnregisterForIdentityUpdatesLocked.
 func (n *NameManager) Lock() {
```

Comment:
@christarazi isn't this redundant? You embed the `lock.RWMutex` which promotes its methods up to the `NameManager` structure, no ?

Reply:
It is, but this code existed before this change and I didn't want to refactor everything else around the mutex when I just wanted to get to the optimization of using a rwmutex.

Reply:
Got it, didn't realize that